modules/gateway/gateway.go

```diff
@@ -2,63 +2,90 @@ package gateway
 
 import (
 	"errors"
+	"net"
 	"sync"
 
 	"github.com/NebulousLabs/Sia/consensus"
 	"github.com/NebulousLabs/Sia/modules"
-	"github.com/NebulousLabs/Sia/network"
+)
+
+const (
+	// maxStrikes is the number of "strikes" that can be incurred by a peer
+	// before it will be removed.
+	// TODO: need a way to whitelist peers (e.g. hosts)
+	maxStrikes = 5
 )
 
 var (
+	// TODO: unexport these, or move them to modules
```

Comment:
I can see them being used by clients, so that you can show the user a special "Your peer list is empty!" dialogue instead of a generic "no peers" error. But those clients generally won't have access to the modules package anyway, so it's a moot point. They don't seem terribly useful for other modules either. I think we should unexport them until we have a compelling reason to do otherwise.